Cubical/Categories/Category.agda

```diff
@@ -24,59 +24,90 @@ module Cubical.Categories.Category where
 open import Cubical.Core.Glue
 open import Cubical.Foundations.Prelude
 
+private
+  variable
+    ℓ ℓ' : Level
+
 -- Precategories
 
 record Precategory ℓ ℓ' : Type (ℓ-suc (ℓ-max ℓ ℓ')) where
   no-eta-equality
   field
     ob : Type ℓ
-    hom : ob → ob → Type ℓ'
-    idn : ∀ x → hom x x
-    seq : ∀ {x y z} (f : hom x y) (g : hom y z) → hom x z
-    seq-λ : ∀ {x y : ob} (f : hom x y) → seq (idn x) f ≡ f
-    seq-ρ : ∀ {x y} (f : hom x y) → seq f (idn y) ≡ f
-    seq-α : ∀ {u v w x} (f : hom u v) (g : hom v w) (h : hom w x) → seq (seq f g) h ≡ seq f (seq g h)
+    Hom[_,_] : ob → ob → Type ℓ'
+    id : ∀ x → Hom[ x , x ]
+    _⋆_ : ∀ {x y z} (f : Hom[ x , y ]) (g : Hom[ y , z ]) → Hom[ x , z ]
+    seq-λ : ∀ {x y : ob} (f : Hom[ x , y ]) → (id x) ⋆ f ≡ f
+    seq-ρ : ∀ {x y} (f : Hom[ x , y ]) → f ⋆ (id y) ≡ f
+    seq-α : ∀ {u v w x} (f : Hom[ u , v ]) (g : Hom[ v , w ]) (h : Hom[ w , x ]) → (f ⋆ g) ⋆ h ≡ f ⋆ (g ⋆ h)
+
+  -- composition: alternative to diagramatic order
+  _∘_ : ∀ {x y z} (g : Hom[ y , z ]) (f : Hom[ x , y ]) → Hom[ x , z ]
+  g ∘ f = f ⋆ g
 
 open Precategory public
 
+
+-- Helpful syntax/notation
+
+_[_,_] : (C : Precategory ℓ ℓ') → (x y : C .ob) → Type ℓ'
+_[_,_] = Hom[_,_]
+
+-- needed to define this in order to be able to make the subsequence syntax declaration
```

Comment:
Could one make these private?

Reply:
I don't think they can be made private since the syntax isn't a standalone thing. I also tried defining the operator itself as _⋆⟨_⟩_ but the issue there is that the category is the second argument, while the first argument is a morphism of that category, but Agda requires that arguments can only depend on previous arguments (is there some way to get around this)?

Reply:
Ok, let's leave it as it is then